### PR TITLE
Fix code analysis on Travis CI

### PR DESCRIPTION
**Pull request by hvelarde**
I found the problem in another place; as you can see this has nothing to do with the lack of `travis.cfg` configuration.

I'll wait for comments on how to proceed; caching in not being used neither.

Refs. #184

### CODE REVIEW

File: addon_test.template.sh [hunk 1]
```diff
@@ -10,14 +10,15 @@ ${buildout:directory}/bin/mrbob -O ${:addon_name} -n bobtemplates:plone_addon --
 cd ${buildout:directory}/${:addon_name}
 virtualenv ${buildout:directory}/${:addon_name}
 ${buildout:directory}/${:addon_name}/bin/python bootstrap-buildout.py --setuptools-version=8.3
-${buildout:directory}/${:addon_name}/bin/buildout
+${buildout:directory}/${:addon_name}/bin/buildout code-analysis:return-status-codes=True
```
Comment (hvelarde) [inline addon_test.template.sh]: this was the problem; the rest is just debug stuff.

Comment (gforcada) [inline addon_test.template.sh]: 👍 so now that you found it, would you mind removing all the debug stuff? we certainly don't want to keep that around :-)

File: bobtemplates/plone_addon/src/+package.namespace+/+package.name+/testing.py.bob [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from plone import api
```
Comment (gforcada) [inline bobtemplates/plone_addon/src/+package.namespace+/+package.name+/testing.py.bob]: Is this also a leftover? 😄

**Comment by hvelarde** [comment]
@tisto now that I found the issue I want to be very clear: **this was broken way before I even touched the package** and had nothing to do with the changes I proposed: if you had looked at it with an open mind you had noticed so since the beginning.

I respect you as a person and as a professional and I learned a lot with you in the past; the same as you, I do testing and code analysis all workdays since many years ago. I proposed those changes not because I wanted to make your life miserable, but because I though they were good, and it happens some people agreed with me.

I can have a good argument with anyone, I have done that in the past with some other folks for months here and in the community forum. I found your comments pretty disrespectful to me and other Plone folks; you're not better than anybody else here, and you're not different.

take care and have a nice weekend; I'll have a beer for the sake of us.

File: bobtemplates/plone_addon/src/+package.namespace+/+package.name+/testing.py.bob (hunk 2 above)
Comment (hvelarde) [inline bobtemplates/plone_addon/src/+package.namespace+/+package.name+/testing.py.bob]: I know, I just want to read some comments before I remove it :-)

**Comment by hvelarde** [comment]
@tisto I still waiting for your comments.

**Comment by tisto** [comment]
@hvelarde sorry, I don't have time right now to look into this (I am about to leave for holidays). Go ahead with whatever you think is appropriate...

**Comment by jensens** [comment]
closed unmerged? Why?

**Comment by tisto** [comment]
@hvelarde do you have time for a video call tomorrow to discuss this issue face to face?

**Comment by hvelarde** [comment]
@tisto sure, I always have time; let me know when it works best for you.

**Comment by tisto** [comment]
@hvelarde what about now? :)

**Comment by tisto** [comment]
@hvelarde actually tomorrow might be better. I will ping you when I'm at the office. Any time tomorrow during the day works for me...

**Pull request merged**